data/zaktech96.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "Zakariye Sahid",
+  "displayStatsPublic":true,
+  "type":"personal",
+  "bio": "Front end developer"
+  "type": "Personal"
+  "avater":"https://github.com/zaktech96.png"
```

We don't need these properties anymore. Please remove these from your `json`! 

```suggestion
  "bio": "Front end developer"
```